integration-tests/__tests__/__snapshots__/failures.test.js.snap

```diff
@@ -118,7 +118,7 @@ exports[`works with async failures 1`] = `
     +   \\"foo\\": \\"bar\\",
       }
       
-      at ../../packages/expect/build/index.js:145:57
+      at ../../packages/expect/build/index.js:104:76
```

Comment:
we should probably remove this line entirely

Reply:
Feel free to remove the lilne:column mapping in a followup